### PR TITLE
Updates example config to match production values

### PR DESCRIPTION
**Pull request by Allfd**
This PR changes the example config settings for gameplay to match the settings used on the production paradise server.

Hopefully this will help reduce expected behavior during testing turning into unexpected behavior during deployment.

Database options are in use on production, however since most developers don't use databases during code development the SQL databases functions remain disabled in the example configs.

### CODE REVIEW

File: config/example/config.txt [hunk 1]
```diff
@@ -77,23 +77,25 @@ KICK_INACTIVE
...
-PROBABILITY NUCLEAR 3
-PROBABILITY ABDUCTION 0
-PROBABILITY CHANGELING 3
-PROBABILITY CULT 4
-PROBABILITY EXTEND-A-TRAITORMONGOUS 5
-PROBABILITY TRAITORCHAN 4
-PROBABILITY TRAITORVAMP 4
-PROBABILITY BLOB 2
+PROBABILITY EXTEND-A-TRAITORMONGOUS 3
+PROBABILITY TRAITORCHAN 3
```
Comment (taukausanake) [inline config/example/config.txt]: "T-Traitor-chan! W-why are you covered in blood?"
"Captain-kun... You wouldn't understand"

File: config/example/config.txt [hunk 2]
```diff
@@ -77,23 +77,25 @@ KICK_INACTIVE
...
 PROBABILITY RAGINMAGES 0
 PROBABILITY METEOR 0
-PROBABILITY TRAITOR 0
-PROBABILITY SHADOWLING 2
+PROBABILITY EXTENDED 1
+PROBABILITY HEIST 0
+PROBABILITY VAMPIRE 3
+PROBABILITY BORER 0
+PROBABILITY XENOS 0
+PROBABILITY MUTINY 0
```
Comment (Fox-McCloud) [inline config/example/config.txt]: Borer, Xenos,  Malfunction, and Mutiny should all be removed as they're not modes anymore.

File: config/example/config.txt [hunk 3]
```diff
@@ -77,23 +77,25 @@ KICK_INACTIVE
 ##
 ## default probablity is 1, increase to make that mode more likely to be picked
 ## set to 0 to disable that mode
-PROBABILITY EXTENDED 2
-PROBABILITY NUCLEAR 3
-PROBABILITY ABDUCTION 0
```
Comment (Fox-McCloud) [inline config/example/config.txt]: This should be added to the live server and left at this value.

File: config/example/jobs.txt [hunk 4]
```diff
@@ -6,32 +6,35 @@ Research Director=1
 Chief Medical Officer=1
 
 Station Engineer=5
-Roboticist=1
+Atmospheric Technician=5
```
Comment (Fox-McCloud) [inline config/example/jobs.txt]: What is this file even used for?

Comment (taukausanake) [inline config/example/jobs.txt]: Number of jobs available at round start? Isn't that handled somewhere else?

File: config/example/config.txt (hunk 3 above)
Comment (Allfd) [inline config/example/config.txt]: Fixed on both

File: config/example/jobs.txt (hunk 4 above)
Comment (Allfd) [inline config/example/jobs.txt]: Does anyone know?  It was in the examples.

Comment (Fox-McCloud) [inline config/example/jobs.txt]: I'm not sure if this is used, at all?

**Pull request merged**